src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -75,13 +76,18 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
     const columnsIndex = [...new Set(cellsPosition.map((p) => p.col))].sort((a, b) => a - b);
     const rowsIndex = [...new Set(cellsPosition.map((p) => p.row))].sort((a, b) => a - b);
 
-    const cellsInClipboard: ClipboardCell[][] = [];
+    const cellsInClipboard: Maybe<ClipboardCell>[][] = [];
     const sheetId = getters.getActiveSheetId();
 
     for (let row of rowsIndex) {
-      let cellsInRow: ClipboardCell[] = [];
+      let cellsInRow: Maybe<ClipboardCell>[] = [];
       for (let col of columnsIndex) {
         const position = { col, row, sheetId };
+        const spreader = getters.getArrayFormulaSpreadingOn(position);
+        if (spreader && rowsIndex.includes(spreader.row) && columnsIndex.includes(spreader.col)) {
+          cellsInRow.push(undefined);
+          continue;
+        }
```

Comment:
if we do that, copy/past cells with their own spreader will not copy/past the style and the border of the spreading cells, no ?